main.go

```diff
@@ -15,29 +15,27 @@ import (
 )
 
 var ( //flags
-	inPath         string
-	outPath        string
-	multi          bool
-	threaded       bool
-	numCertThreads int
-	prettyPrint    bool
-	numProcs       int
-	channelSize    int
+	inPath           string
+	outPath          string
+	numCertThreads   int
+	prettyPrint      bool
+	numProcs         int
+	channelSize      int
+	crashIfParseFail bool
 )
 
 func init() {
-	flag.StringVar(&inPath, "in-file", "", "File path for the input certificate(s).")
-	flag.StringVar(&outPath, "out-file", "-", "File path for the output JSON.")
+	flag.StringVar(&inPath, "input-file", "", "File path for the input certificate(s).")
+	flag.StringVar(&outPath, "output-file", "-", "File path for the output JSON.")
 	flag.BoolVar(&prettyPrint, "list-lints-json", false, "Use this flag to print supported lints in JSON format, one per line")
-	flag.BoolVar(&multi, "multi", false, "Use this flag to specify inserting many certs at once. Certs in this mode must be Base64 encoded DER strings, one per line.")
-	flag.BoolVar(&threaded, "threads", false, "Use this flag to specify that -multi mode runs multi-threaded. This has no effect otherwise.")
 	flag.IntVar(&numCertThreads, "cert-threads", 1, "Use this flag to specify the number of threads in -threads mode.  This has no effect otherwise.")
-	flag.IntVar(&numProcs, "procs", 1, "Use this flag to specify the number of processes to run on.")
+	flag.IntVar(&numProcs, "procs", 0, "Use this flag to specify the number of processes to run on.")
 	flag.IntVar(&channelSize, "channel-size", 10000, "Use this flag to specify the number of values in the buffered channel.")
+	flag.BoolVar(&crashIfParseFail, "crash", false, "Use this flag if you want to crash on parsing failure.")
```

Comment:
How about a more descriptive name and an a declarative (imperative?) description? Maybe `--fatal-parse-errors`?